context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This programming language is for artists. It allows artists to create complex pattern art with thousands of repeated shapes that might otherwise be tedious and time consuming to create.
+
```

Comment:
Agreed, think about the different features of the language and what audience they might be designed for.